FoodieTestCase/FoodieTest.cs

```diff
@@ -0,0 +1,89 @@
+using System;
+using Xunit;
+using Foodie;
+
+namespace FoodieTestCase
+{
+    public class FoodieTest
+    {
+        [Fact]
+        public void AddFood()
+        {
+            Food pizza = new Pizza();
+            MyCart cart = new MyCart();
+            ViewCart viewCart;
+
+            cart.Add(pizza, 3);
+            viewCart = new ViewCart(cart.GetCurrentCart());
+
+            Assert.Equal(3, viewCart.GetQuantity(pizza));
+        }
+
+        [Fact]
+        public void TestTotalPrice()
```

Comment:
Test method names should be more descriptive that demonstrates the scenario well.